FE/src/features/stories/ui/Card/Card.tsx

```
@@ -1,19 +1,43 @@
 import Link from 'next/link';
+import type { StoriesCard } from '@/features/stories/model/stories.type';
+import { useState } from 'react';
+
+interface StoriesCardProps {
+  id: string;
+  story: StoriesCard;
+}
+
+const DEFAULT_THUMBNAIL = '/FE/src/assets/NoImage.png';
 
 const StoriesCard = ({ id, story }: StoriesCardProps) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'StoriesCard'. Consider to delete it or rename it.

'StoriesCard' is defined here:


(lint/suspicious/noRedeclare)